backend/test/db.test.ts

```diff
@@ -42,9 +45,14 @@ describe('db init', () => {
         };
     }
 
+    function getDbModuleWithMockedLogger(dbConfig: DbConfig, mockedLogger: ReturnType<typeof mockLogger>) {
+        // @ts-ignore Passing mocked logger intentionally
+        return getDbModule(dbConfig, mockedLogger, [() => model]);
```

Comment:
If I just have `<LoggerFactory>mockedLogger` I'll get the following error:
```
TS2352: Conversion of type '{ getLogger: () => { info: (...args: any[]) => void; error: (...args: any[]) => void; debug: jest.Mock ; }; logErrorsOnly: (...args: any[]) => void; }' to type 'LoggerFactory' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

I can use the same solution as above: `<LoggerFactory>(<unknown>mockedLogger)`. What do you think?

BTW I introduced `getDbModuleWithMockedLogger` to minimize the number of casting/ignores.